libs/ui/date-picker/helm/src/lib/hlm-date-format.token.ts

```diff
@@ -0,0 +1,25 @@
+import { inject, InjectionToken, ValueProvider } from '@angular/core';
+
+export interface HlmDatePickerConfig<T> {
+	/**
+	 * Defines how the date should be displayed in the UI.
+	 *
+	 * @param date
+	 * @returns formatted date
+	 */
+	formatDate: (date: T) => string;
+}
+
+const defaultConfig: HlmDatePickerConfig<Date> = {
+	formatDate: (date) => (date instanceof Date ? date.toDateString() : `${date}`),
+};
```

Comment:
Not sure if the default should have `Date` as type, because a different `BrnDateAdapter` can be provided (like `BrnLuxonDateAdapter` or custom)